plugins/modules/one_image.py

```diff
@@ -114,300 +107,432 @@
 id:
     description: image id
     type: int
-    returned: success
+    returned: when O(state=present, state=cloned, state=renamed)
```

Comment:
```suggestion
    returned: when O(state=present), O(state=cloned), or O(state=renamed)
```
Same for the others.

(Otherwise you're saying that this is returned if `state` has the value `present, state=cloned, state=renamed`, which isn't a valid value :) )

Reply:
yes, I was unsure which one is right, thanks for clarifying :)